### DISCUSSION TITLE
Improvements, new stuff and a Discord server

### DISCUSSION THREAD

**Discussion by rednblkx** [rednblkx/HomeKey-ESP32#67] [Announcements]
I'll start this off by announcing that you can now find attached on these releases firmware binaries, you can find more information about them in the [wiki](https://github.com/rednblkx/HomeKey-ESP32/wiki/Setup#2-flashing).

- Implemented a simple web interface that includes all options previously in `config.h` and more
- This release includes the new PN532 SPI Driver that was reworked to use ESP-IDF SPI Master APIs and some other improvements, this should solve the "Didn't find PN53x board" that can appear at times
- Added option to trigger a GPIO Pin on HomeKey Success or Fail, should address #57
- Added option to trigger a GPIO Pin when interacting with the Lock from the Home App
- Improvements and optimizations
- Tested on ESP32-C3 and works about the same (firmware binaries attached below)

Additionally i have now launched a Discord server called "HomeKit Corner"(might rename it to "HomeKit Matters" or something to include Matter), this is where current and future projects will be discussed. This should help streamline communication and make it easier to answer questions and solve any issues, if you have any of that or just want to hangout you can join the server at the link below.

https://discord.gg/VWpZ5YyUcm

<hr /><em>This discussion was created from the release <a href='https://github.com/rednblkx/HomeKey-ESP32/releases/tag/v0.3'>Improvements, new stuff and a Discord server</a>.</em>

**Comment by moshiko2312** [comment]
Sounds good thanks
